unified-codes/apps/web/.storybook/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
Looking back I think I was getting some warnings and made this change, can remove if it was just me

Reply:
Already done in a different PR so this is gonezers after merge